client/OPUNetGameSelectWnd.cpp

```diff
@@ -542,7 +542,7 @@ void OPUNetGameSelectWnd::OnReceiveHostedGameSearchReply(Packet& packet)
 	// Check for allocation errors
 	if (hostedGameInfo == nullptr)
 	{
-		SetStatusText("Out of memory");
+		LogDebug("Out of memory");

```

Comment:
I'm kind of undecided if this one might be worth informing the user about. At any rate, it's quite unlikely to ever be triggered, so it likely doesn't matter much. Up to your discretion.